pkg/interface/chat/src/js/components/new.js

```diff
@@ -33,6 +34,27 @@ export class NewScreen extends Component {
         props.history.push('/~chat/room' + station);
       }
     }
+
+    if (prevState.security !== this.state.security) {
+
+      let securityText = '';
+
+      switch (this.state.security) {
+        case 'village':
+          securityText = 'Invite-only chat. Default membership administration.';
+          break;
+        case 'channel':
+          securityText = 'Completely public chat. Default membership administration.';
+          break;
+        case 'journal':
+          securityText = 'Similar to a blog. Publicly readable/subscribable, invited members can write to journal.'
+          break;
+        case 'mailbox':
```

Comment:
Let's comment out mailboxes from the list of options for the time being. User stories haven't been adequately worked out.

Reply:
Added